cifar10-multiannotator/utils/model_training.py

```diff
@@ -0,0 +1,106 @@
+import sys
+
+sys.path.insert(0, "../")
+
+import numpy as np
+import pandas as pd
+import pickle
+import datetime
+from pathlib import Path
+import cleanlab
+from .cross_validation_autogluon import cross_val_predict_autogluon_image_dataset
```

Comment:
How about just moving the `cross_val_predict_autogluon_image_dataset` function into this file as well?  So that users only need to consider one .py file of helper functions beyond the example notebook.